week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,11 @@
+fetch("https://xkcd.now.sh/?comic=latest")
+  .then((response) => response.json())
+
+  .then((data) => {
+    let imageTag = document.createElement("img");
+    console.log(data);
+    imageTag.src = data.img;
+    document.body.appendChild(imageTag);
```

Comment:
hear you got it the other way around 😃 , they wanted it to change each time. 
but good work. Try to avoid appending anything to the body object. As you won't do in real coding

Reply:
Added error handling here as well, but here API gives only one object and that is not randomly changing each time unless i am missing something.